environments/prod/backend_lb_config.yaml

```diff
@@ -199,3 +199,10 @@ gateways:
  # FACT
       - product: fact
         component: api
+
+      # Docmosis
+      - product: dg
+        component: docmosis
+        cookie_based_affinity: Enabled
```

Comment:
ew

Reply:
Yeah, it's actually needed only for the UI (which rarely people use as its horrible). But BA's seems to use it just to verify if updates went fine.